docs/docs/how-tos/nebari-stages-directory.md

```diff
@@ -0,0 +1,46 @@
+---
+id: nebari-stages-directory
+title: Nebari Stages Directory
+description: Purpose and Usage of the "stages" Directory
+---
+
+# The Nebari "Stages" Directory
+
+The Nebari CLI generates Terraform manifests and saves them on disk in directories `stages/[stage-name]` relative to the location of your configuration file.  Knowing how and when these directories are used will allow you to better track changes that the Nebari CLI makes to your running Nebari deployment when you upgrade or modify the configuration file.
+
+## Nebari CLI Steps
+The following three Nebari CLI commands trigger different steps in the Nebari deployment process.
+
+`nebari validate` will check whether your configuration file is valid for the Nebari schema of the Nebari CLI version that you are using.  It will **NOT** render new `stages` directories.
+
+`nebari render` will validate your configuration file **AND** generate new Terraform manifests in the `stages` directories but will **NOT** deploy those changes.
+
+`nebari deploy` will validate your configuration file, generate new Terraform manifests **UNLESS** you use the `--disable-render`, **AND** deploy those changes.  If `--disable-render` is set, the Nebari CLI will instead use whatever code is present in the existing `stages` directory.  This is useful to ensure that only Terraform code changes that you have seen are present with the deploy (see Tip below).
```

Comment:
There is another one that might be helpful displaying here as we are already talking about the extension below, `nebari info` shows all current "installed" stages (as they are also plugins) currently in place -- e.g that will be rendered/deployed:
<img width="1045" alt="Captura de Tela 2024-07-02 às 19 51 51" src="https://github.com/nebari-dev/nebari-docs/assets/51954708/baf3f92f-4d72-45f4-99fe-072831c390e7">

Reply:
Done